Doc/library/curses.ascii.rst

```diff
@@ -176,7 +176,7 @@ C library:
 
    Checks for a non-ASCII character (ordinal values 0x80 and above).
 
-These functions accept either integers or strings; when the argument is a
+These functions accept either integers or single-character strings; when the argument is a
```

Comment:
Should this be changed? It accept multi-character strings, too. Alought it will raise `TypeError` by `ord`.

Also, after this paragraph, it has written this `Note that all these functions check ordinal bit values derived from the first character of the string you pass in`.